hack/operatorhub/internal/container/container.go

```diff
@@ -220,7 +220,7 @@ func getFirstUndeletedImage(images []Image) *Image {
 // container image locally, and push it to Quay.io registry using the
 // provided credentials.
 func pushImageToRegistry(c PushConfig) error {
-	if c.DryRun {
+	if c.DryRun && !c.Force {
```

Comment:
Why did you change this? Isn't it strange to do `--dry-run --force`?

Reply:
Because if the same image already exists, you can't force-push the image, or add latest tag, but that doesn't exactly make sense with dry-run, agreed. I've removed this. If dry-run is set it should never do anything.
```
	log.Printf("Determining if image already exists within project with tag: %s", c.Tag)
	exists, err := ImageExistsInProject(c.HTTPClient, c.RedhatCatalogAPIKey, c.ProjectID, c.Tag)
	if err != nil {
		log.Println("ⅹ")
		return fmt.Errorf("failed to determine if image exists: %w", err)
	}
	if exists && !c.Force {
		log.Println("✓")
		log.Println("not continuing as image was already found within redhat project")
		return nil
	}
	if c.Force {
		log.Println("pushing image as force was set")
	}

	if err = pushImageToRegistry(c); err != nil {
		log.Println("x")
		return fmt.Errorf("failed to push image: %w", err)
	}
````